test_cases/TestCase_11.json

```diff
@@ -149,15 +109,29 @@
     ],
     "preconditions": [],
     "trapi_template": null,
-    "components": [
-        "ars"
-    ],
     "test_case_objective": "AcceptanceTest",
     "test_case_source": null,
     "test_case_predicate_name": "treats",
     "test_case_predicate_id": "biolink:treats",
     "test_case_input_id": "MONDO:0018958",
-    "test_case_runner_settings": [
-        "inferred"
-    ]
+    "qualifiers": [
+        {
+            "parameter": "biolink_qualified_predicate",
+            "value": "biolink:treats"
+        },
+        {
+            "parameter": "biolink_object_aspect_qualifier",
+            "value": ""
+        },
+        {
+            "parameter": "biolink_object_direction_qualifier",
+            "value": ""
+        }
+    ],
+    "input_category": "biolink:Disease",
+    "output_category": "biolink:ChemicalEntity",
```

Comment:
Can it always be expected that the output category be the same across test case assets? I see it as adding specificity for the query template used, but would there ever be a case where the outputs are different categories? I think I could imagine a test template that ends up sending NamedThing as the output category?

Reply:
That would definitely change the way we generate the TestCases.  This PR doesn't restrict us from adding another way of making TestCases where the TestCase input/output categories don't match the TestAsset input/output categories.